reduce.js

```diff
@@ -8,3 +8,11 @@ var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
 // via multiplication
 
 
+
+const multiply = (acc, next) => {
+    return acc*next
+  }
```

Comment:
nice! Question for you to answer for yourself: what's the accumulator if you don't name it explicitly?